starter-code/scripts/articleView.js

```diff
@@ -53,13 +73,25 @@ articleView.setTeasers = function() {
   // Truncate logic to show only first two elements within the article body.
   $('.article-body *:nth-of-type(n+2)').hide();
   /* TODO: Add a delegated event handler to reveal the remaining paragraphs.
-    When a .read-on link is clicked, we can:
-    1. Prevent the default action of a link.
-    2. Reveal everything in that particular article now.
-    3. Hide that read-on link!
+  When a .read-on link is clicked, we can:
+  1. Prevent the default action of a link.
+  2. Reveal everything in that particular article now.
+  3. Hide that read-on link!
 
-    // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
+  // STRETCH GOAl!: change the 'Read On' link to 'Show Less'
   */
+  $('article').on('click', 'a[href="#"]', function(event) {
+    event.preventDefault();
+    console.log($(event.target).parent());
```

Comment:
So this isn't exactly what you want to do. Instead of focusing on the event, you should focus on the specific element that you are trying to manipulate. Hint: starts with an R and ends with an "on" :)